lib/Sanger/CGP/CavemanPostProcessing/Flagger.pm

```diff
@@ -38,12 +39,28 @@ use Sanger::CGP::CavemanPostProcessing;
 const my $BAD_BITS => hex RFLAGS->{UNMAPPED}|RFLAGS->{M_UNMAPPED}|RFLAGS->{NOT_PRIMARY}|RFLAGS->{QC_FAILED}|RFLAGS->{DUPLICATE}|RFLAGS->{SUPPLEMENTARY};
 const my $GOOD_BITS => hex RFLAGS->{MAP_PAIR};
 
+const my $VCF_COLUMN_NORMAL => 'NORMAL';
+const my $VCF_COLUMN_TUMOUR => 'TUMOUR';
+const my $VCF_COLUMN_FORMAT => 'FORMAT';
+const my $OLD_ALLELE_VCF_FORMAT => 'GT:AA:CA:GA:TA:PM';
+const my $NEW_ALLELE_VCF_FORMAT => 'GT:FAZ:FCZ:FGZ:FTZ:RAZ:RCZ:RGZ:RTZ:PM';
+const my %OLD_ALLELE_VCF_FORMAT_INDEX_HASH => ('A' => 1, 'C' => 2, 'G' => 3, 'T' => 4, );
+const my %NEW_ALLELE_VCF_FORMAT_INDEX_HASH => ('A'=>[1,5], 'C' =>[2,6], 'G'=>[3,7], 'T'=>[4,8], );
+
+const my $MATCH_CIG => 'M';
+const my $SKIP_CIG => 'N';
+const my $INS_CIG => 'I';
+const my $DEL_CIG => 'D';
+const my $SOFT_CLIP_CIG => 'S';
+const my $HARD_CLIP_CIG => 'H';
```

Comment:
These same constants are in `lib/Sanger/CGP/CavemanPostProcessing.pm` and should be moved out into a constants class.  Remember you can do:

```perl
use Sanger::CGP::CavemanPostProcessing::Const;
const my $C => 'Sanger::CGP::CavemanPostProcessing::Const';
# localise for speed (far faster than going through the sub everytime)
const my $MATCH_CIG => $C->MATCH_CIG;
const my $SKIP_CIG => $C->SKIP_CIG;
...
```

To do this you have to define them as subs but it is clean.

Reply:
Need to compare new and old and ensure same results then get rid of old code base as new one is far more flexible imho. Not for this feature though. Agree constants need organising. Is that for this feature or a refactoring release?

Reply:
WRT constants it depends how much code is affected, if they're only used in a couple of places they could be done now, completely your choice.  At least setup a milestone with tasks for anything we've discussed.  We can then present it as cleanup that needs to be done to make this more maintainable.

Reply:
Removed 'new' unused flagging code to new feature branch.